### ISSUE TITLE
Renaming a collection doesn't remove old schedule

### ISSUE THREAD

**Issue by ComLock** [enonic/app-explorer#324]

**Comment by sigdestad** [comment]
Would it be a better solution to collection id rather than name for task? Harder to detect tasks in a low-level view ofc...

**Comment by ComLock** [comment]
Currently we are using both...

```
const jobPrefix = `${collectorId.replace(':', DOT_SIGN)}${DOT_SIGN}${collectionNodeName}${DOT_SIGN}`;
```

https://github.com/enonic/lib-explorer/blob/master/src/main/resources/lib/explorer/scheduler/createOrModifyJobsFromCollectionNode.es#L57

```
name: `${jobPrefix}${i}`
```

https://github.com/enonic/lib-explorer/blob/master/src/main/resources/lib/explorer/scheduler/createOrModifyJobsFromCollectionNode.es#L115

**Comment by ComLock** [comment]
Misread, you said collectION id, not collectOR id.

The problem with _id is that it is NOT guaranteed to be unique cross repo.
Thus we decided to use strings (from _name) as identifiers.
However we should have kept using _id within the same repo, and this has been gradually fixed along the way.
For example interfaces still uses collectionName rather than collectionId.

I did suggest a cross repo identifyer: `${repoId}.${nodeId}`, but that was turned down...

But something like that seems to be the perfect solution in this instance.
To make it possible to search for collectionName in the system.scheduler repo, maybe it can be added to the config (if it's not already there).
